web/src/settings_config.ts

```diff
@@ -872,14 +872,18 @@ export const realm_name_in_email_notifications_policy_values = {
 export const desktop_icon_count_display_values = {
     messages: {
         code: 1,
-        description: $t({defaultMessage: "All unreads"}),
+        description: $t({defaultMessage: "All unread messages"}),
```

Comment:
Hmm, this might be more accurately described as "All unmuted unread messages"; we don't offer an "All unread messages including muted ones" option.